src/shared/components/schoolCard/schoolCard.css

```diff
@@ -2,10 +2,10 @@
   display: flex;
   border-radius: 4px;
   align-items: center;
-  margin: 20px 10px;
+  margin: 25px 10px;
```

Comment:
It seems fine in chrome, so maybe I can just change most of it back and add some media queries. What about being able to click on the card other that just the title? I feel like that's a keeper.